src/components/pages/questions/questions.js

```diff
@@ -84,30 +86,12 @@ class Questions extends Component {
           }, 100)
         }
       }
-      if( input === 'totalMortgagesBalanceForLender') {
-        let elem = document.getElementById('totalMortgagesBalanceForLender')
-        if(elem && elem.value) {
-          elem.value = this.numberWithCommas(elem.value)
-        }
-      }
-      if( input === 'propertyEstimatedRentalIncome') {
-        let elem = document.getElementById('rentalIncome')
-        if(elem && elem.value) {
-          elem.value = this.numberWithCommas(elem.value)
-        }
-      }
-      if( input === 'propertyValue') {
-        let elem = document.getElementById('propertyValue')
+      inputsWithCommas.forEach( id => {
```

Comment:
document.querySelectorAll('.normalise').forEach( elem => {
if (elem.value) {
          elem.value = this.numberWithCommas(elem.value)
        }
} )

Reply:
Agreed.